azure-pipelines/linux-coverage.yaml

```diff
@@ -8,9 +8,10 @@ jobs:
       pool:
           name: Azure Pipelines
           demands: java
-          vmImage: "ubuntu-18.04"
+          vmImage: "ubuntu-20.04"
```

Comment:
can we just use `ubuntu-latest`?

Reply:
We can - I just didn't want the pipeline to suddenly start failing if it updates. Then again, if we suddenly start failing on the latest Ubuntu, we probably want to know, so I'll change that.